02-express-tutorial/app.js

```diff
@@ -10,7 +10,7 @@ app.get('/api/v1/test', (req, res) => {
 
 //Route to get a product by ID
 app.get('/api/v1/products/:productID', (req, res) => {
-    const { productID } = req.params;
+    const { products } = req.params;
```

Comment:
Do we have a `products` on our `req.params`?

The parameters come from the `:` that we use on line 12 in `/api/v1/products/:productID`, so, we only have that `productID` available in our `req.params`.